test/functorch/test_aotdispatch.py

```diff
@@ -1905,10 +1905,11 @@ def forward(self, x):
                 # constant to make_fx, and result in the tensor being traced
                 # into the graph, which is an error condition.  Make sure we
                 # report adequately in this case.
-                return (x + fake_z, )
+                return (torch.add(x, fake_z), )
 
         with self.assertRaisesRegex(
-            AssertionError, "Unexpected fake buffer"
+            TypeError,
+            "no implementation found for .*FakeTensor"
```

Comment:
This is a much harder error for the user but maybe it's also on them for using fake tensor directly

Reply:
ok, I can think of some possibilities for making this error easier, but this is fine. (Probably easiest is to have some logging when we report NotImplemented, and then have the handler say how to turn the logging on.)

Reply:
A more invasive API approach would be to make NotImplemented sigils also sideband report why they were NotImplemented, so we can collect these up and tell the user about it.